index.js

```diff
@@ -1,88 +1,135 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
 
-// console.log(counter) 
+// const counter = 2
+
+// var counter = 1
+// var counter = 2
+
+let counter = 1
+counter = 2
```

Comment:
Great! The key here was using `const` because it doesn't allow us to reassign our variable